src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/_help.py

```diff
@@ -280,3 +280,21 @@
     type: command
     short-summary: Lists the firewall rules.
     """
+helps['sql server vnet-rule'] = """
+    type: group
+    short-summary: Manage a server's virtual network rules.
+    """
+helps['sql server vnet-rule update'] = """
+    type: command
+    short-summary: Updates a virtual network rule.
+    """
+helps['sql server vnet-rule create'] = """
+    type: command
+    short-summary: Creates a virtual network rule which allows access to an Azure Sql server.
+
+    examples:
```

Comment:
Nice examples, thank you! I love what you did here with the ID and name options!